modules/gapi/perf/cpu/gapi_core_perf_tests_cpu.cpp

```diff
@@ -160,6 +160,12 @@ INSTANTIATE_TEST_CASE_P(SumPerfTestCPU, SumPerfTest,
         //Values(0.0),
         Values(cv::compile_args(CORE_CPU))));
 
+INSTANTIATE_TEST_CASE_P(CountNonZeroPerfTestCPU, CountNonZeroPerfTest,
+    Combine(Values(AbsToleranceScalar(0.0).to_compare_f()),
```

Comment:
You had to fight with the misalignment, but not join it.)

Reply:
This was intentional as all other tests in this file are not aligned. But.. I mused about it more and now I think you are right. But I won't align other tests not to complicate the diff (and review process) :)